onnx/defs/schema.h

```diff
@@ -388,6 +388,11 @@ class OpSchema final {
                                       : dummyInferenceFunction;
   }
 
+  OpSchema& PartialDataPropagationFunction(DataPropagationFunction partialDataProgationFunction);
```

Comment:
minor nit: perhaps we can just call it ```DataPropagationFunction``` ... shorter, and consistent with the getter function below.